src/components/Custom/InfoButton/InfoButton.js

```diff
@@ -18,14 +19,12 @@ function InfoButton(props) {
   const [showConfirmDelete, setShowConfirmDelete] = useState(false);
   const { customReport } = props;
 
-  const httpHeaders = Object.assign(
-    {},
-    {
-      'X-Okapi-Tenant': props.stripes.okapi.tenant,
-      'X-Okapi-Token': props.stripes.store.getState().okapi.token,
-      'Content-Type': 'application/json',
-    }
-  );
+  const httpHeaders = {
+    'X-Okapi-Tenant': props.stripes.okapi.tenant,
+    ...getLegacyTokenHeader(props.stripes.okapi),
+    'Content-Type': 'application/json',
+    credentials: 'include',
+  };
```

Comment:
`credentials` should not pe a part of the http header, instead it should be a request option for `fetch`. See this [example](https://github.com/folio-org/ui-invoice/blob/3554dca8fc0d1ac4ae11ed93f0d6f559e102bf02/src/invoices/InvoiceForm/utils.js#L33-L44).
This applies to multiple locations in this PR.

Reply:
@alb3rtino  Thanks for your feedback! Moved the credentials outside of the header.